src/autosize.js

```diff
@@ -55,6 +55,8 @@ function assign(ta) {
 
 		if (style.boxSizing === 'content-box') {
 			heightOffset = -(parseFloat(style.paddingTop)+parseFloat(style.paddingBottom));
+		} else if (style.boxSizing === 'border-box') {
+			heightOffset = -(parseFloat(style.borderTopWidth) + parseFloat(style.borderBottomWidth));
```

Comment:
Maybe remove the spacing around the `+` to be consistent with the other statements